ckanext/push_errors/tests/test_utils.py

```diff
@@ -0,0 +1,91 @@
+import logging
+from ckanext.push_errors.utils import get_error_trace_id, get_error_log_id
+
+
+class CustomError(Exception):
+    pass
+
+
+def raise_custom_error():
+    """Raises a CustomError to simulate an error scenario for testing purposes."""
+    raise CustomError("Simulated error")
+
+
+def test_get_error_trace_id_returns_path_and_line():
+    """
+    Test that get_error_trace_id returns a string containing the file path and line number
+    where the CustomError was raised, formatted as 'path/to/file.py:<line>'.
+    Ensures the returned trace_id contains a colon and does not end with '.py'.
+    """
+    try:
+        raise_custom_error()
+    except CustomError as e:
+        trace_id = get_error_trace_id(e)
+        assert ":" in trace_id
+        assert not trace_id.endswith(".py")  # it only ends with ":<line>"
```

Comment:
```suggestion
        assert trace_id == 'test_utils.py:25'
```
Change to something like this
Use a really specifi value if possible